packages/frontend-2/.env.example

```diff
@@ -42,3 +42,5 @@ NUXT_PUBLIC_ENABLE_DIRECT_PREVIEWS=true
 
 # Ghost API
 NUXT_PUBLIC_GHOST_API_KEY=
+
+NUXT_WEBFLOW_API_TOKEN=8c9bea4c120742a21ff308cda0bea73f13e89ffe26dcc886990bba353549b652
```

Comment:
Thanks for your comments. In this case, it shouldn't be NUXT_PUBLIC_ because it's a server-side API route (it's in fe2/server/api). Server routes can access all environment variables directly.

On the first point, this is not meant to be secret. It only has read access to blog stories, and it has already been added to the Connectors repo in plain text